src/tools/wdump/wdump.cpp

```diff
@@ -0,0 +1,4142 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D dump tool
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "always.h"
+#include "bufffileclass.h"
+#include "chunkio.h"
+#include "resource.h"
+#include "simplevec.h"
+#include "vector.h"
+#include "vector2.h"
+#include "vector3.h"
+#include "vector3i.h"
+#include "w3d_file.h"
+#include "w3d_obsolete.h"
+#include <commctrl.h>
+#include <commdlg.h>
+#include <unordered_map>
+
+HWND mainwnd = nullptr;
+HWND treewnd = nullptr;
+HWND listwnd = nullptr;
+HMENU menu = nullptr;
+HACCEL accel = nullptr;
+int mainwidth = 0;
+int mainheight = 0;
```

Comment:
I suggest name these globals with `g_` prefix too for visibility.

Reply:
fixed